paddle/fluid/framework/details/reference_count_pass.cc

```diff
@@ -267,6 +191,99 @@ static bool ShrinkNoNeedBufferVarOpDependency(
   }
 }
 
+/**
+ * Find the nearest downstream computation op handle. If the op is a
+ * computation op, just return itself.
+ */
+static ComputationOpHandle *FindNextComputationOpHandleOrReturnItself(
+    OpHandleBase *op, size_t scope_idx) {
+  std::queue<OpHandleBase *> q;
+  std::unordered_set<OpHandleBase *> visited;
+  q.push(op);
+  do {
```

Comment:
Prefer "while" loop than "do while" loop. 

There is q.push(op) before the while loop. It's no harm to write
while (!q.empty()) {
    ...
}

Reply:
Done.